scipy-1.0/covlett.tex

```diff
@@ -0,0 +1,89 @@
+% based on template:
+% https://tex.stackexchange.com/questions/15532/looking-for-cover-letter-template
+
+\documentclass[10pt,stdletter,dateno]{newlfm}
+
+% to adjust vertical spacing properties of
+% newlfm class see: https://tex.stackexchange.com/a/65533/43006
+\topmarginsize{2mm}
+\topmarginskip{0in}
+\headermarginsize{0in}
+\headermarginskip{0in}
+
+\usepackage{kpfonts}
+\usepackage{url}
+
+\widowpenalty=1000
+\clubpenalty=1000
+
+\newlfmP{headermarginskip=10pt}
+\newlfmP{sigsize=30pt}
+\newlfmP{dateskipafter=20pt}
+%\newlfmP{addrfromphone}
+%\newlfmP{addrfromemail}
+%\PhrPhone{Phone}
+%\PhrEmail{Email}
+
+\namefrom{Ralf Gommers,
+          Matt Haberland,
+          and Tyler Reddy}
+\addrfrom{%
+    \today\\[10pt]
+    Quansight Labs,\\
+    The Netherlands\\
+    \\
+    BioResource and Agricultural Engineering,\\
+    California Polytechnic State University,\\
+    San Luis Obispo, CA, 93407, USA\\
+    \\
+    CCS-7 Applied Computer Science Group,\\
+    Los Alamos National Laboratory,\\
+    Los Alamos, NM, 87545, USA
+}
+
+\addrto{%
+Prof. Syma Khalid,\\
+School of Chemistry,\\
+University of Southampton,\\
+Highfield Campus,\\
+Southampton,\\
+SO17 1BJ, U.K.}
+
+\greetto{Dear Prof. Khalid,}
+\closeline{Sincerely,}
+\begin{document}
+\begin{newlfm}
+
+We are submitting the enclosed manuscript describing a milestone
+1.0 release of SciPy for consideration to be published in
+\emph{Scientific Reports}. SciPy is the fundamental scientific 
+computing library in the Python programming language. 
+Hundreds of experts have contributed to our library, which
+has been downloaded millions of times and has over 100,000 dependent
+code repositories.
+
+We have specifically chosen \emph{Scientific Reports} because
+the breadth of acceptable topics matches the variety of algorithms
+our library provides. High-profile manuscripts citing SciPy range
+from the biosciences to recent reports on gravitational waves and
+black holes. These citations target a website, and we now think
```

Comment:
"These citation target a website, and we now think it is well past time..."  This latter parts reads a bit strangely to me, but my brain doesn't have an alternative to offer right now.

Reply:
Something like this? 

> Since these citations, for lack of a better alternative, cite the website that SciPy currently located, we would like to provide an academic article for this milestone which also signals better the SciPy's maturity level.

I ran out of English towards the end though.

Reply:
Ah, that gives me a good start!

> Lacking a better alternative, these citations all point to the SciPy website; instead, we think the package now (would be better served | merits | deserves) a formal publication in a highly visible journal.

Reply:
Ok, I've applied Stefan's suggested revision of this sentence